packages/data/src/types.d.ts

```diff
@@ -2,31 +2,31 @@ export type WPDataFunctionOrGeneratorArray = Array< Function | Generator >;
 export type WPDataFunctionArray = Array< Function >;
 
 export interface WPDataAttachedStore {
```

Comment:
Is it using now different indentation than JS? Can we align?

Reply:
This looks correct to me @gziolo, the indentation is changes spaces to tabs, I suspect this had been missed by ESLint/Prettier at some point previously:

<img width="611" alt="image" src="https://user-images.githubusercontent.com/1016458/100951883-930bf180-3563-11eb-85cd-92da5a04cdc2.png">

Reply:
Yes, this file previously used spaced and was merely reformatted to tabs. There's been no radical change to how TS files are formatted and there are no differences between JS an TS formatting.

Reply:
Now I noticed that `tsc` uses 4 spaces indentation when generating the `*.d.ts` files in `build-types`. That's probably where this formatting originated. And Babel apparently indents with 2 spaces.

Reply:
I was browsing this file on mobile, I could verify whether it's tabs or spaces, it looked like 2 spaces. Ignore this comment One tab is correct.